pluginlib/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ install(
 if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
 
-  if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_FLAGS MATCHES "-stdlib=libc\\+\\+")
+  if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND 9 VERSION_LESS_EQUAL CMAKE_CXX_COMPILER_VERSION)
```

Comment:
`VERSION_LESS_EQUAL` doesn't exist in CMake 3.5 which is the minimum version of this CMake file.

Reply:
Thanks! good catch! This seems like an ideal thing for CI to be checking...